src/main/java/com/catcher/batch/core/converter/CatcherConverter.java

```diff
@@ -0,0 +1,42 @@
+package com.catcher.batch.core.converter;
+
+import com.catcher.batch.core.dto.MovieApiResponse;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.json.JSONObject;
+
+public class CatcherConverter<T> {
+    private final String headerPath;
+    private final ObjectMapper objectMapper;
+    private final Class<T> clazz;
```

Comment:
```suggestion
    private final Class<T> responseType;
```
더 명확하게 어떤것에 대한 변수인지 해두는 차원에서? 괜찮은것 같아요